worker/internal/usecase/gateway/cms.go

```diff
@@ -1,7 +1,11 @@
 package gateway
 
-import "context"
+import (
+	"context"
+
+	"github.com/reearth/reearth-cms/server/pkg/asset"
+)
 
 type CMS interface {
-	NotifyAssetDecompressed(ctx context.Context, assetID string) error
+	NotifyAssetDecompressed(ctx context.Context, assetID string, status *asset.Status) error
```

Comment:
Don't import server module in worker module. Do not complicate dependencies easily. Define asset.Status in the worker package

Reply:
Done!